<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ficheMobilite</class>
 <widget class="QDialog" name="ficheMobilite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_sauvegarderMobilite">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimerMobilite">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bloquedMobilite">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Debloquer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dureeControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>279</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenomControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_annee">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>249</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_duree">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>279</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenom">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>57</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_commentaireControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_duree">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>284</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duree</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_anneeControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>249</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ecole">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>145</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ecole</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_commentaire">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>308</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_pays">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>201</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pays</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_annee">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>254</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Annee</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_classeControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>87</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_paysControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ecoleControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>141</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_diplomeControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>169</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_commentaire">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>318</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Commentaire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_diplome">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>173</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diplome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>27</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nomControl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>31</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_classe">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>92</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Classe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenom">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>62</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>32</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_classe">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ecole">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_diplome">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pays">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nom</tabstop>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>lineEdit_classe</tabstop>
  <tabstop>lineEdit_ecole</tabstop>
  <tabstop>lineEdit_diplome</tabstop>
  <tabstop>lineEdit_pays</tabstop>
  <tabstop>lineEdit_annee</tabstop>
  <tabstop>lineEdit_duree</tabstop>
  <tabstop>plainTextEdit_commentaire</tabstop>
  <tabstop>pushButton_sauvegarderMobilite</tabstop>
  <tabstop>pushButton_supprimerMobilite</tabstop>
  <tabstop>pushButton_bloquedMobilite</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
